core/commands/files/files.go

```diff
@@ -235,14 +235,33 @@ Examples:
 			return
 		}
 
+		long, _, _ := req.Option("l").Bool()
```

Comment:
i think it is already one -- https://github.com/ipfs/go-ipfs/pull/2007/files#diff-1feff21640e90e71a5ec9ccec2650fbdR217